sysutils/AutoRaise/Portfile

```
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        sbmpost AutoRaise 1.5 v
+
+categories          sysutils
+platforms           macosx
+
+license             {GPL-2 GPL-3}
+maintainers         {@lhaeger gmx.net:lothar.haeger} openmaintainer
+
+description         AutoRaise raises application windows to the front and gives them focus on mouse hover
+long_description    ${description} after a configurable delay. There is also an option to warp the mouse \
+                    to the center of the activated window, using the cmd-tab key combination for example.
+
+homepage            htps://github.com/sbmpost/AutoRaise
+
+checksums           rmd160  6569b47fe2570fe8f079ea06c96c43314a9eb4df \
+                    sha256  a42fe824a3e1f8cb51529b52321ba138bef966a5fa496ac538c466046ced3969 \
+                    size    28613
+
+use_configure       no
+
+build.cmd           g++
```

[Use the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler), please.
```suggestion
build.cmd           ${configure.cxx}
```